libraries/botbuilder-dialogs-adaptive/src/actions/actionScope.ts

```diff
@@ -61,7 +61,7 @@ export class ActionScope<O extends object = {}> extends Dialog<O> implements Dia
 
     /**
      * Called when the dialog is started and pushed onto the dialog stack.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.